build.gradle

```diff
@@ -36,6 +36,18 @@ subprojects {
 	}
 }
 
+allprojects {
+	plugins.withType(JavaPlugin).whenPluginAdded {
+		dependencies {
+			constraints {
+				implementation("com.amazonaws:aws-xray-recorder-sdk-core:2.18.2") {
+					because "older versions are incompatible with Jackson >= 2.15"
+				}
+			}
+		}
+	}
+}
```

Comment:
This is a transitive dependency from AWS Lambda powertools, and the older versions use some features that have been removed in recent versions of Jackson.

There is a planned 'v2' of powertools that might bring us back up to date, but they're otherwise not doing any updates to 'v1' (beyond critical fixes).